tools/wds2idx.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python3
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is the license valid?

Reply:
I copied it directly from the rec2idx file, I can swap it out for a proper one though

Reply:
done